packages/vite/src/node/server/index.ts

```diff
@@ -115,6 +115,12 @@ export interface ServerOptions extends CommonServerOptions {
    * @default true
    */
   preTransformRequests?: boolean
+  /**
+   * Whether or not to ignore-list source files in the dev server sourcemap, used to populate
+   * the [`x_google_ignoreList` source map extension](https://developer.chrome.com/blog/devtools-better-angular-debugging/#the-x_google_ignorelist-source-map-extension).
+   * By default, it excludes all paths containing `/node_modules/`.
+   */
+  sourcemapIgnoreList?: (sourcePath: string) => boolean
```

Comment:
To make it 100% compatible with the predicate in rollup, should we also pass the `sourcemapPath` as second parameter? It'd basically be `mod.file + ".map"` below.

Reply:
I initially didn't do this as we are passing the absolute path to via `sourcePath`, whereas in rollup we get a relative path. (But maybe it will also be relative once https://github.com/vitejs/vite/pull/12079 merges?)

Reply:
Depending on the order. But I'd think it's safer to just pass absolute paths to both `sourcePath` and `sourcemapPath` here.